src/main/io/serial.h

```diff
@@ -49,6 +49,7 @@ typedef enum {
     FUNCTION_VTX_TRAMP           = (1 << 13), // 8192
     FUNCTION_RCDEVICE            = (1 << 14), // 16384
     FUNCTION_LIDAR_TF            = (1 << 15), // 32768
+    FUNCTION_FRSKY_OSD           = (1 << 20), // 1048576
```

Comment:
Why not 16?

Reply:
[16, 19] are already used in INAV. I thought that having the same FUNCTION id could be preferable in both firmwares, but using different ones is not a big problem either.

Reply:
The values < 16 are not in synch between iNav and Betaflight already, so I don't think there is not much value in having this one in synch.